model/permission.go

```diff
@@ -557,6 +558,12 @@ func initializePermissions() {
 		"authentication.permissions.read_channel.description",
 		PermissionScopeChannel,
 	}
+	PermissionReadTopic = &Permission{
+		"read_topic",
+		"",
+		"",
```

Comment:
Should we define these pre-emptively, even if we're not exposing it in the system console yet?